sdk/program/src/wasm/pubkey.rs

```diff
@@ -49,7 +50,8 @@ impl Pubkey {
                 }
                 return Err(format!("Invalid array argument: {:?}", x).into());
             }
-            Ok(Pubkey::new(&bytes))
+            Pubkey::try_from(Pubkey::try_from(bytes))
+                .map_err(|err| Err(format!("Invalid pubkey: {err:?}")))
```

Comment:
we could improve DX by disambiguating these two error messages.  maybe add the input type?

Reply:
what is "DX"?
added input type to error messages.

Reply:
> what is "DX"?

developer experience

> added input type to error messages.

thanks!